config/opal_setup_zlib.m4

```diff
@@ -27,25 +29,29 @@ AC_DEFUN([OPAL_ZLIB_CONFIG],[
     if test "$with_zlib" != "no"; then
         AC_MSG_CHECKING([for zlib in])
         if test ! -z "$with_zlib" && test "$with_zlib" != "yes"; then
+            AC_MSG_RESULT([$with_zlib])
```

Comment:
I don't think this should be here as you have AC_MSG_RESULT outputs below that are otherwise unmatched

Reply:
Probably should add opal_zlib_standard_lib_location and opal_zlib_standard_header_location to the list of "pushed" variables

Reply:
i fixed that in #4422, and will cherry-pick once merged after CI completes
Thanks !